lib/subdomains/base.rb

```diff
@@ -0,0 +1,9 @@
+module Subdomains
```

Comment:
Why you paste it in `lib` folder? In `lib` folder usually stored vendor code libraries. All code that you write should be in `app` folder